lib/project_types/script/messages/messages.rb

```diff
@@ -5,9 +5,9 @@ module Messages
     MESSAGES = {
       script: {
         error: {
-          deprecated_ep: "This project uses an extension point %s which has been deprecated. "\
+          deprecated_ep: "This project uses the %s Script API which has been deprecated. "\
```

Comment:
Nitpick: should it be "Scripts API": https://shopify.dev/beta/scripts/scripts-apis
Also, isn't this sentence a bit wordy?  Can it just be "This project uses a deprecated API."?